client/src/styles/_chosen.scss

```diff
@@ -3,3 +3,8 @@
 .chosen-container-multi .chosen-choices {
   padding: 4px 5px;
 }
+
+// Avoid chosen dropdowns overflowing when not active
```

Comment:
Could we add a comment that this is only necessary for browsers other than Firefox? I think the original issue was that the SS devs thought that they didn't need this anymore because they were likely using firefox.

Reply:
The comment should be written and positioned in a way that clearly covers any changes which work towards resolving this bug, since this bug as a whole only affects non-firefox.

Reply:
I've added comments. Still not sure what the intend of #1422 was. Guy you mentioned it probable was to fix a bug with fluent locale switcher. I've tried to reproduce but couldn't. The fluent-switch is a native dropdown no chosen-thingy AFAICT.